pkg/util/netutil/local_ip.go

```diff
@@ -34,3 +36,8 @@ func LocalIP() (localhost net.IP) {
 
 	return
 }
+
+func RandomLoopbackAddr() net.IP {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function RandomLoopbackAddr should have comment or be unexported